intake/gui/source/defined_plots.py

```diff
@@ -4,14 +4,27 @@
 #
 # The full license is in the LICENSE file, distributed with this software.
 #-----------------------------------------------------------------------------
-from copy import deepcopy
+from distutils.version import LooseVersion
+
+try:
+    import dfviz
+    assert LooseVersion(dfviz.__version__) >= LooseVersion("0.1.0")
```

Comment:
Confused by this, the highest version of dfviz seems to be 0.0.3, so this isn't allowing me to display the GUI at all.

Reply:
https://anaconda.org/conda-forge/dfviz/files version is 0.1.0

Reply:
Ah, thanks, I just cloned the repo.